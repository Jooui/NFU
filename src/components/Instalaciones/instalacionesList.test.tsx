import React from 'react';
import renderer from 'react-test-renderer';
import InstalacionesList from './instalacionesList';

it('renders correctly', () => {
  const tree = renderer
    .create(<InstalacionesList></InstalacionesList>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});