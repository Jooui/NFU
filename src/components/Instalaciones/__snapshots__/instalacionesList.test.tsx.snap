// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ion-list>
  <ion-card
    href="/app/instalacion/1"
    onClick={[Function]}
    router-link="/app/instalacion/1"
  >
    <section
      className="cardContent"
    >
      <article
        className="cardContent__left"
      >
        <img
          alt="img"
          src="https://estaticos-cdn.prensaiberica.es/clip/6a4607f2-8199-44f3-ac8a-68eee7a21561_16-9-discover-aspect-ratio_50p_0.jpg"
        />
      </article>
      <article
        className="cardContent__right"
      >
        <ion-card-title>
          Polideportivo Ontinyent
        </ion-card-title>
        <ion-card-subtitle>
          <ion-icon
            icon="data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' class='ionicon' viewBox='0 0 512 512'><title>Pin</title><path d='M339 99a83 83 0 10-102 80.8V464l19 32 19-32V179.8A83.28 83.28 0 00339 99zm-59-6a21 21 0 1121-21 21 21 0 01-21 21z'/></svg>"
          />
          Ontinyent,Valencia
        </ion-card-subtitle>
        <ion-card-content>
          <ion-icon
            icon="data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' class='ionicon' viewBox='0 0 512 512'><title>Heart</title><path d='M352.92 80C288 80 256 144 256 144s-32-64-96.92-64c-52.76 0-94.54 44.14-95.08 96.81-1.1 109.33 86.73 187.08 183 252.42a16 16 0 0018 0c96.26-65.34 184.09-143.09 183-252.42-.54-52.67-42.32-96.81-95.08-96.81z' stroke-linecap='round' stroke-linejoin='round' class='ionicon-fill-none ionicon-stroke-width'/></svg>"
          />
               
          <ion-icon
            icon="data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' class='ionicon' viewBox='0 0 512 512'><title>Enter</title><path d='M176 176v-40a40 40 0 0140-40h208a40 40 0 0140 40v240a40 40 0 01-40 40H216a40 40 0 01-40-40v-40' stroke-linecap='round' stroke-linejoin='round' class='ionicon-fill-none ionicon-stroke-width'/><path stroke-linecap='round' stroke-linejoin='round' d='M272 336l80-80-80-80M48 256h288' class='ionicon-fill-none ionicon-stroke-width'/></svg>"
          />
        </ion-card-content>
      </article>
    </section>
  </ion-card>
  <ion-card
    href="/app/instalacion/2"
    onClick={[Function]}
    router-link="/app/instalacion/2"
  >
    <section
      className="cardContent"
    >
      <article
        className="cardContent__left"
      >
        <img
          alt="img"
          src="http://www.xativa.es/sites/default/files/images/SeccioDeports.jpg"
        />
      </article>
      <article
        className="cardContent__right"
      >
        <ion-card-title>
          Polideportivo Xativa
        </ion-card-title>
        <ion-card-subtitle>
          <ion-icon
            icon="data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' class='ionicon' viewBox='0 0 512 512'><title>Pin</title><path d='M339 99a83 83 0 10-102 80.8V464l19 32 19-32V179.8A83.28 83.28 0 00339 99zm-59-6a21 21 0 1121-21 21 21 0 01-21 21z'/></svg>"
          />
          Xativa,Valencia
        </ion-card-subtitle>
        <ion-card-content>
          <ion-icon
            icon="data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' class='ionicon' viewBox='0 0 512 512'><title>Heart</title><path d='M352.92 80C288 80 256 144 256 144s-32-64-96.92-64c-52.76 0-94.54 44.14-95.08 96.81-1.1 109.33 86.73 187.08 183 252.42a16 16 0 0018 0c96.26-65.34 184.09-143.09 183-252.42-.54-52.67-42.32-96.81-95.08-96.81z' stroke-linecap='round' stroke-linejoin='round' class='ionicon-fill-none ionicon-stroke-width'/></svg>"
          />
               
          <ion-icon
            icon="data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' class='ionicon' viewBox='0 0 512 512'><title>Enter</title><path d='M176 176v-40a40 40 0 0140-40h208a40 40 0 0140 40v240a40 40 0 01-40 40H216a40 40 0 01-40-40v-40' stroke-linecap='round' stroke-linejoin='round' class='ionicon-fill-none ionicon-stroke-width'/><path stroke-linecap='round' stroke-linejoin='round' d='M272 336l80-80-80-80M48 256h288' class='ionicon-fill-none ionicon-stroke-width'/></svg>"
          />
        </ion-card-content>
      </article>
    </section>
  </ion-card>
  <ion-card
    href="/app/instalacion/3"
    onClick={[Function]}
    router-link="/app/instalacion/3"
  >
    <section
      className="cardContent"
    >
      <article
        className="cardContent__left"
      >
        <img
          alt="img"
          src="https://lolleria.org.es/wp-content/uploads/2017/04/poliesportiu_municipal_olleria1-2.jpg"
        />
      </article>
      <article
        className="cardContent__right"
      >
        <ion-card-title>
          Polideportivo Olleria
        </ion-card-title>
        <ion-card-subtitle>
          <ion-icon
            icon="data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' class='ionicon' viewBox='0 0 512 512'><title>Pin</title><path d='M339 99a83 83 0 10-102 80.8V464l19 32 19-32V179.8A83.28 83.28 0 00339 99zm-59-6a21 21 0 1121-21 21 21 0 01-21 21z'/></svg>"
          />
          L'Olleria,Valencia
        </ion-card-subtitle>
        <ion-card-content>
          <ion-icon
            icon="data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' class='ionicon' viewBox='0 0 512 512'><title>Heart</title><path d='M352.92 80C288 80 256 144 256 144s-32-64-96.92-64c-52.76 0-94.54 44.14-95.08 96.81-1.1 109.33 86.73 187.08 183 252.42a16 16 0 0018 0c96.26-65.34 184.09-143.09 183-252.42-.54-52.67-42.32-96.81-95.08-96.81z' stroke-linecap='round' stroke-linejoin='round' class='ionicon-fill-none ionicon-stroke-width'/></svg>"
          />
               
          <ion-icon
            icon="data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' class='ionicon' viewBox='0 0 512 512'><title>Enter</title><path d='M176 176v-40a40 40 0 0140-40h208a40 40 0 0140 40v240a40 40 0 01-40 40H216a40 40 0 01-40-40v-40' stroke-linecap='round' stroke-linejoin='round' class='ionicon-fill-none ionicon-stroke-width'/><path stroke-linecap='round' stroke-linejoin='round' d='M272 336l80-80-80-80M48 256h288' class='ionicon-fill-none ionicon-stroke-width'/></svg>"
          />
        </ion-card-content>
      </article>
    </section>
  </ion-card>
</ion-list>
`;
